modules/datastore/src/Drush.php

```diff
@@ -52,8 +52,6 @@ class Drush extends DrushCommands {
 
   /**
    * The PostImportResultFactory service.
-   *
-   * @var \Drupal\datastore\PostImportResultFactory
```

Comment:
I'm surprised CC didn't flag this and the other @var removals. Can we modify rector so it leaves 'unneeded' @vars behind? I guess we also need to talk about coding standards in general and tooling in particular.

Reply:
ok. no biggie.